test/e2e/usersignup_test.go

```diff
@@ -45,6 +45,55 @@ func (s *userSignupIntegrationTest) TearDownTest() {
 	memberAwait2.Clean(s.T())
 }
 
+func (s *userSignupIntegrationTest) TestUserSignupMigration() {
+	hostAwait := s.Host()
+	memberAwait := s.Member1()
+	// Create user signup
+	hostAwait.UpdateToolchainConfig(s.T(),
+		testconfig.AutomaticApproval().Enabled(true),
+		testconfig.CapacityThresholds().
+			MaxNumberOfSpaces(
+				testconfig.PerMemberCluster(memberAwait.ClusterName, 1000),
+			).
+			ResourceCapacityThreshold(80))
```

Comment:
just curious - if there was any specific reason for setting those values. As defaults would be `MaxNumberOfSpaces: 0` (which means infinity) and `ResourceCapacity` is already 80 by default. So maybe those are not needed, but it's a minor thing.

Reply:
Good catch, this was a copy paste from another test and the intention wasn't to set this config to anything out of the ordinary, thanks.